src/dictionaries/en.ts

```diff
@@ -18,7 +18,8 @@ export const dictionary = {
     ALERT_DIALOG_DEFAULT_CANCEL_BUTTON: 'Cancel',
     ALERT_DIALOG_DEFAULT_CONTINUE_BUTTON: 'Continue',
     ALERT_DIALOG_DEFAULT_SUBMIT_BUTTON: 'Submit',
-    ALERT_DIALOG_DEFAULT_CONFIRM_BUTTON: 'Confirm'
+    ALERT_DIALOG_DEFAULT_CONFIRM_BUTTON: 'Confirm',
+    ALERT_DIALOG_LINK_TO_PULL_REQUEST_BUTTON: 'Link to Pull Request'
```

Comment:
Which is better use 'DEFAULT' or not? 😅
I thought LINK_TO_REQUEST_BUTTON is not default so I didn't use it.